backend/canvas_app_explorer/views.py

```diff
@@ -1,15 +1,33 @@
-from django.shortcuts import render
-
+from canvas_oauth.oauth import get_oauth_token
+from django.conf import settings
 from rest_framework import permissions, viewsets
+from rest_framework.request import Request
 
 from backend.canvas_app_explorer import models, serializers
-
+from .canvas_lti_manager import CanvasLtiManager
 
 class LTIToolViewSet(viewsets.ModelViewSet):
     """
-    API endpoint that allows users to be viewed or edited.
+    API endpoint that lists LTI tools available in the course context
     """
-    queryset = models.LtiTool.objects.all()
     serializer_class = serializers.LtiToolSerializer
+    permission_classes = [permissions.DjangoModelPermissions]
+    queryset = models.LtiTool.objects.all()
+
+    def get_serializer_context(self):
+        context = super(LTIToolViewSet, self).get_serializer_context()
+        request: Request = context['request']
+        access_token = get_oauth_token(request)
+        manager = CanvasLtiManager(
```

Comment:
We probably could simplify this `CanvasLtiManager` so you only need to call it with 2 parameters most of the time. (Or all of the time)

So you'd be like `CanvasLtiManager(api_key=access_token, course_id=request.session['course_id']`). If could just pull that `api_url` parameter directly.

Actually, we could probably also just change `CanvasLtiManger` to just take a single parameter request. Then it would be even less code outside. I just didn't see how it was going to be used yet.